<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PrOplataButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Проверить
 сумму</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_dollar.png</normaloff>../images for project/icon_dollar.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="ItogoLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Итого: </string>
    </property>
   </widget>
   <widget class="QLabel" name="SummaLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Minibar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>309</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пользование минибара</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SpaSalon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>309</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пользование Спа</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Fitnes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>309</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пользование Фитнес-залом</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OplataButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Провести
 оплату</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/oplata_icon.png</normaloff>../images for project/oplata_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>378</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
